org.openfodder.OpenFodder.yml

```diff
@@ -0,0 +1,65 @@
+app-id: org.openfodder.OpenFodder
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.llvm11
+build-options:
+  append-path: /usr/lib/sdk/llvm11/bin
+  prepend-ld-library-path: /usr/lib/sdk/llvm11/lib
+command: openfodder
+rename-appdata-file: openfodder.appdata.xml
+rename-desktop-file: openfodder.desktop
+rename-icon: openfodder
+finish-args:
+  - --socket=wayland
```

Comment:
does it really support wayland?

Reply:
I don't know. Can't test as Wayland is effectively broken on my machines. It uses SDL2.

Reply:
The controls are also a bit wonky on X11.

Reply:
Dropped Wayland.